src/components/SearchBoxMobile.js

```diff
@@ -0,0 +1,73 @@
+import SearchIcon from '@mui/icons-material/Search';
+import {
+  Box,
+  Fab,
+  IconButton,
+  InputAdornment,
+  InputBase,
+  TextField,
+} from '@mui/material';
+import { makeStyles } from 'models/makeStyles';
+import React from 'react';
+
+const useStyles = makeStyles()((theme) => ({
```

Comment:
New components should ideally only use the `sx` prop for consistency and access to it's advanced features. You also left unused style classes that are cluttering the file. Move the styles into `sx` prop objects and delete this useStyles hook.

Reply:
@QuaidBartolomei thanks for the feedback. I have updated the component as per your suggestion. See the snap of the code below. Regarding the screenshot you mentioned, I used the result from running cypress component tester. I am not sure why the padding is not working on your side. Can I ask why where Figma would be useful in this case?

![image](https://user-images.githubusercontent.com/61099792/147858724-26b79299-ad8f-4f8c-ba35-00510ecb3dc2.png)

Reply:
Currently below is how the search box looks.

![image](https://user-images.githubusercontent.com/61099792/147858841-4af24dd0-bb64-4a45-aae4-43bc94735119.png)

Reply:
The component file looks much better. Next time you should push your changes so that I can properly review them.

> I am not sure why the padding is not working on your side. Can I ask why where Figma would be useful in this case?

It's not about if it renders for me locally, I based my comment off of your screenshot. It is important to provide an accurate screenshot. The Figma document allows you to inspect the padding values.

```
borderRadius: '5vh',
paddingLeft: '5vh'
```
These should be fixed values, the figma doc should be used for matching these values

Reply:
Thanks. I did not commit the changes yet because I wanted to confirm my fix was right. I am completely new to Figma. Can you explain how that can be used to determine the styling values? May be if you have time today we can go over it quickly on slack.

Reply:
@dagmawig You will only need to use Figma for inspecting the design so it should be simple. Here is the Figma doc: https://www.figma.com/file/XdYFdjlsHvxehlrkPVYq0l/Greenstand-Webmap

Make sure you are signed in so that you have a view similar to this: 
![2022-01-02 13 11 57 www figma com c2b29743d29a](https://user-images.githubusercontent.com/59486980/147889627-4454b34e-bbd9-47cf-8812-80d8e1e572d7.png)

- Use the left section to select the mobile designs.
- Use middle section to find the correct component (use ctrl + mouse wheel to zoom, and space + click to pan). 
- After clicking on a component you can use the right section to inspect its style settings. You can also mouse over other components to measure the distance between the edges of that component and the component currently selected.

Let me know if you need more specific directions for any of this.

>  I did not commit the changes yet because I wanted to confirm my fix was right.

You can commit anyways. It may clutter the commit history, but it's worth it to help the review process. In the worst case, the commit history could always be squashed with an interactive rebase.

Reply:
Great! I just made the styling update as per the doc and made the commit. Let me know if it needs any further changes.

> You can commit anyways. It may clutter the commit history, but it's worth it to help the review process. In the worst case, the commit history could always be squashed with an interactive rebase.

I will do that going forward.